tailwind.config.js

```diff
@@ -7,6 +7,11 @@ module.exports = {
         'sprite-background': "url('/images/icons.png')",
       })
     },
+    colors: {
+      violet: {
+        200: "#f4dcf4"
```

Comment:
set theme variable name as `primary-color`

Reply:
so that tomorrow we can decide to use another color without having to change its usage at all places